.github/workflows/copy-workflow.yml

```diff
@@ -14,10 +14,11 @@ jobs:
         workflow: ${{ github.event.client_payload.workflows }}
     env:
       WORKFLOW_DIR: "workflow-repo"
-      FILE: "${{ matrix.workflow }}.yml"
+      TEMPLATE_DIR: "templates"
+      FILE: "${{ matrix.workflow }}"
```

Comment:
should all those other references to "workflow" or "WORKFLOWS" also be replaced with a generic "file" or "FILES"?

Reply:
Yes, that makes sense.